xugrid/ugrid/ugrid1d.py

```diff
@@ -289,6 +289,13 @@ def coords(self):
             self.edge_dimension: self.edge_coordinates,
         }
 
+    @property
+    def length(self):
```

Comment:
I think this still needs a test?

Reply:
Good catch, added test, and fixed bug which showed up during this.